modules/GOTOPGO.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.preview.dsl=2
+
+process goTopGo {
+  input:
+    file DESeq2.tsv
+  output:
+    file topGO_GO_up.txt
+    file topGO_GO_dn.txt
+  script:
+    """
+    ../bin/GOTOPGO.R DESeq2.tsv
```

Comment:
You not need to specify the path to the R script if it is located in `bin`:

`GOTOPGO.R ${DESeq2_in}`

should be fine